.github/workflows/newdeploy.yml

```
@@ -0,0 +1,352 @@
+            cp .env .github/scripts/ansible/docker
+
+  
+
+
+  deploy:
+    needs: [ print_timestamp, build-go-binary ]
+    runs-on: self-hosted
+
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

actionlint (1.7.4)

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)